redhat-cop/casl-ansible
# Introduce Ansible Galaxy to handle repo dependencies

#### What does this PR do?
This changes the setup of the CASL repo to handle the inter-repo dependencies. I.e.: in this first iteration, it pulls in dependent roles from `infra-ansible` as part of the setup prep for running CASL.

#### How should this be manually tested?
Follow the README file and ensure the `ansible-galaxy` step is executed before running the provisioning. Run the provisioning and ensure everything works as it used to. 

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl
